### ISSUE TITLE
[BUG]Index Page Exception When Model No `created_at` Column

### ISSUE THREAD

**Issue by luonet** [avo-hq/avo#434]
### Describe the bug
If we have a ActiveRecord model without `created_at` column,
then when we go to the resource index page

An exception will raise. For example:
`SQLite3::SQLException: no such column: genres.created_at`

I checked the avo gem source code, and found it assumed the `created_at` exists:
```ruby
# ruby/3.0.0/lib/ruby/gems/3.0.0/gems/avo-1.8.1/app/controllers/avo/base_controller.rb:222L
@index_params[:sort_by] = params[:sort_by] || :created_at
```

```ruby
# ruby/3.0.0/lib/ruby/gems/3.0.0/gems/avo-1.8.1/app/controllers/avo/base_controller.rb:39L
if @index_params[:sort_by].present?
  @query = @query.order("#{@resource.model_class.table_name}.#{@index_params[:sort_by]} #{@index_params[:sort_direction]}")
end
```

When you use the default model generator of rails, it does have the `created_at` column,
but for many other use cases, for example, a static table, we do not need a `created_at` or `updated_at` column.

So I think it's better to remove this dependency on `created_at`.

### Steps to Reproduce
1. Create a new model without timestamps, such as `rails g model movie name:string --no-timestamps`
2. Run `rake db:migrate`
3. Generate the related avo resource: `rails g avo:resource movie`
4. Run `rails s`, then go to localhost:3000/avo/resources/movies, you'll see the error.

### System configuration
**Avo version**:
1.8.1

**Rails version**:
6.1.3.2

**Ruby version**:
3.0.0

### Additional context
When I comment out the `created_at` in gem source, everything goes back:
```ruby
# ruby/3.0.0/lib/ruby/gems/3.0.0/gems/avo-1.8.1/app/controllers/avo/base_controller.rb:222L
@index_params[:sort_by] = params[:sort_by]# || :created_at
```

So I think it's better to just remove the `created_at` dependency.

**Comment by adrianthedev** [comment]
Mmmyeah. wrong assumption on our part there. I'll push a fix today.

**Comment by adrianthedev** [comment]
This fix is out with the latest [`1.8.2`](https://rubygems.org/gems/avo/versions/1.8.2) version
